livereveal/main.js

```diff
@@ -8,6 +8,19 @@
 * -----------------------------------------------------------------------------
 */
 
+define(['jquery',
+        'base/js/utils',
+        'services/config',
+], function($, utils, configmod) {
+
+var config_section = new configmod.ConfigSection('livereveal',
+                            {base_url: utils.get_body_data("baseUrl")});
+config_section.load();
+var config = new configmod.ConfigWithDefaults(config_section, {
+    theme: 'simple',
+    transition: 'linear'
```

Comment:
we should eventually set up others config options... but that's for another PR...